simulations/simulation_runner.py

```diff
@@ -61,9 +59,13 @@ def step(self):
 
     def _send_messages_along_paths(self, message_paths):
         sent_messages = {}
-        # Send most recent message of sender to receive
         for sender, receiver in message_paths:
-            message = sender.my_latest_message()
+            last_message = sender.my_latest_message()
```

Comment:
don't all validator's have a latest message when they are initialized now? Do we need this check?